src/utils/breadcrumbs.ts

```diff
@@ -12,7 +12,7 @@ export function getRegistryDashCrumb (): BreadcrumbI {
   const account = useBcrosAccount()
   return {
     text: ref(t('breadcrumb.registryDashboard')),
-    href: `${useRuntimeConfig().public.authWebURL}account/${account.currentAccount.id}/business`
+    href: `${useRuntimeConfig().public.businessRegistryURL}account/${account.currentAccount.id}`
```

Comment:
Now, this link will go straight to the BRD (instead of Auth Web which redirects to the BRD if you're logged in).

<img width="291" height="61" alt="image" src="https://github.com/user-attachments/assets/1e2b29be-e76a-453e-9cb2-22453f38904a" />